dynamic_programming/Bellman-Ford.cpp

```diff
@@ -22,7 +22,7 @@ class Graph
 	{
 		this->vertexNum = V;
 		this->edgeNum = E;
-		this->edges = (Edge *)malloc(E * sizeof(Edge));
+		this->edges = (Edge *)malloc(E * sizeof(Edge)); //allocate a block of uninitialised memory
```

Comment:
Code Documentation

Reply:
You may consider looking to the compilation log of failed cases and changing the Documentation of the Code

Reply:
cpplint_modified_files is failing below.